### DISCUSSION TITLE
A mod to only allow alternating

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#13003] [Ideas]
I wanted to switch to full-alt playstyle in taiko while I'm still relatively new, so I modified the game to only allow alternating hands. I was wondering if that kind of feature would be useful as an official mod because usually switching to full-alt is quite painful. Also I've heard always alternating in std helps you with accuracy or something like that so the mod could work for std as well.

**Comment by Tanza3D** [comment]
I like the sound of this.

**Comment by voidedWarranties** [comment]
planning on implementing this, but unsure of details on how full-alt taiko gameplay works. do you remember which hand you use between color changes? after you hit a large note?
should there be options for both of these?

**Comment by chayleaf** [comment]
You switch hand on every note. Most people reset hand after large notes, I personally hit them the same way I'd hit two notes at the same time. I reset after swells and most drumrolls.
